index.Rmd

```diff
@@ -284,13 +288,13 @@ In order to achieve these objectives, you should try to follow a few good practi
     - Indentation (like in Python: use spaces or tab to indent code in functions or loops etc.).
     - Consistent use of the assignment operator: either `<-` or `=` in all your code. The former is used by core R and allows assigning in function calls, the latter is shorter and consistent with most other programming languages.
     - Consistent placement of curly braces.
-* [Make your own packages](#__optional__Writing_packages_).
-* Keep a similar [directory structure](../RProjectManagement/index.html) across your projects.
-* Use [version control](../RProjectManagement/index.html) to develop/maintain your projects and packages.
+* [Make your own packages](#__optional__Writing_packages_). 
+* Keep a similar [directory structure](../RProjectManagement/index.html) across your projects. 
+* Use [version control](../RProjectManagement/index.html#version-control) to develop/maintain your projects and packages.
 * Never place `rm(list = ls())` anywhere in your script. If you do so, you may [find your computer set on fire](https://www.tidyverse.org/blog/2017/12/workflow-vs-script/).
 * Use relative file paths and never use `setwd()`. Do press the button below every time you open a script, and assume everyone else will do that too.
 
-![Session -> Set working directory -> To Source File Location](figs/setwd_rstudiotip.png)
+![Session -> Set working directory -> To Source File Location](figs/setwd_rstudiotip.png) 
```

Comment:
Yes, but better using the top menu, rather than the files tab. This is because you don't want to set random directories as working directories, you want to set the working directory to source file (i.e. main.R) location specifically.